docs/architecture/decisions/0007-add-fact-check-view-as-a-rails-engine.md

```diff
@@ -0,0 +1,87 @@
+# 7. Add Fact Check App as a Rails Engine
+
+Date: 2025-11-18  
+Status: Accepted
+
+## Context
+
+GOV.UK’s Content Block Manager requires a workflow for 2i and Fact Check processes, allowing subject matter experts to 
+review proposed changes to content blocks before publication. These fact checkers often work outside the core GOV.UK 
+organisation and therefore do not have access to GOV.UK Signon.
+
+We need a simple and secure way to share proposed changes with fact checkers so they can quickly view and approve 
+updates. The main application currently provides no mechanism for unauthenticated or externally authenticated access.
+
+## Problem
+
+Fact checkers need read-only access to proposed content block changes, but:
+
+- they cannot use Signon,
+- they must receive links that can be shared outside GOV.UK organisations,
+- the main application should not expose these views directly, and
+- any access mechanism must minimise the security risk to GOV.UK systems and data.
+
+## Decision
+
+We will implement a dedicated, lightweight "fact check" interface as a **Rails engine** within the Content Block Manager 
+application.
+
+The engine will:
+
+- expose read-only views of proposed content block changes,
```

Comment:
is this something like a diff or side-by-side (before and after) views of a rendered content block? i.e. 

1. the "latest published edition" and
2. some other unpublished edition, e.g. a draft?

Reply:
Good plan, I've added that